spec/fibonacci_spec.rb

```diff
@@ -3,16 +3,15 @@
 SimpleCov.start
 
 require_relative '../lib/fibonacci'
-require_relative '../lib/helper'
 
 describe Fibonacci do 
-    let(:helper){Helper.new}
-    subject(:fibonacci) {described_class.new(helper)}
+    subject(:fibonacci) {described_class.new}
 
     it 'it should take an integer and return the fibonacci sequence up to that integer'do 
```

Comment:
You've changed your test (and the Fibonacci kata - from returning the sequence to returning the Fibonacci number), but the test description stayed the same.